permutive/index.js

```diff
@@ -88,10 +96,14 @@ const setPageMetaData = (content, user) => {
 			industry: user.industry && user.industry.code,
 			responsibility: user.responsibility && user.responsibility.code,
 			position: user.position && user.position.code,
+			subscriptionLevel: user.subscriptionLevel,
+			loggedIn: (!!user.loggedInStatus).toString(),
+			indb2b: user.hui.indb2b,
```

Comment:
Is user.hui always defined?

Reply:
Yes, always defined. If it doesn't contain any datapoints it would be an empty object.
This might change in the future though, so I agree it's better to check. I'm going to make the changes

Reply:
Resolved in 5749c5b